.github/workflows/release.yml

```diff
@@ -154,7 +154,7 @@ jobs:
         run: docker load -i cheqd-node-image.tar
 
       - name: Build
-        run: docker build -f docker/single_image_testnet/Dockerfile --no-cache -t cheqd-testnet .
+        run: docker build -f docker/testnet/Dockerfile --no-cache -t cheqd-testnet .
```

Comment:
This will be moved into tests directory and renamed as @ankurdotb suggested to `docker-testnet`. We definitely shouldn't cal it `testnet`.

Reply:
Talked with @ankurdotb and have already reverted it back to single_image_testnet to pick up localnet as a separate PR.